packages/manager/apps/ips/public/translations/onboarding/Messages_fr_FR.json

```diff
@@ -1,13 +1,19 @@
 {
-  "title": "ips",
-  "description": "Découvrez des services de stockage managés qui s’appuient sur le système de fichiers OpenZFS. Bénéficiez en quelques clics d’espaces de stockage centralisés pour entreposer ou sauvegarder vos données et fichiers.",
+  "title": "Additional IP",
+  "titlebis": "Ajoutez de la flexibilité à vos applications avec Additional IP",
+  "description": "La plupart des produits OVHcloud incluent une adresse IPv4 statique pré-configurée. Pour une flexibilité exceptionnelle, ces produits peuvent également être utilisés avec des Additional IP.",
+  "descriptionBis": "Vous pouvez attribuer une Additional IP (adresse individuel ou bloc) à tout service compatible au sein du même campus. Un campus est un groupe de datacenters partageant le même emplacement géographique. Pour plus d'informations veuillez consulter notre page dédiée :",
+  "ip_onboarding_content_list_item_1": "Adresse individuelle (/32)",
+  "ip_onboarding_content_list_item_2": "Bloc d’adresse IP (jusqu’à /24)",
   "orderButtonLabel": "Commander un ips",
+  "orderButtonLink": "/#/dedicated/ip/order",
+  "moreInfoButtonLink": "/#/dedicated/ip/info",
```

Comment:
You can remove those keys
You have to use navigate to go to the pages since they are internal to the app
```
() => navigate(urls.order);
```

Reply:
i cannot just past the navigate function  as prop to the  OnboardingLayout component so i need to create a custom Button for this like this ?
`
      <OdsButton
        className="mb-5"
        variant={ODS_BUTTON_VARIANT.outline}
        icon={ODS_ICON_NAME.plus}
        onClick={() => navigate(urls.order)}
        label={t('orderIps')}
      />
`

Reply:
No there are props for the onClick on each buttons